packages/suite/src/reducers/suite/modalReducer.ts

```diff
@@ -97,4 +99,16 @@ const modalReducer = (state: State = initialState, action: Action): State => {
     }
 };
 
+export const selectTxStakeName = (state: AppState) => {
+    if (
+        state.modal.context === MODAL.CONTEXT_USER &&
+        state.modal.payload.type == 'transaction-detail'
+    ) {
+        const txName = state.modal.payload.tx.ethereumSpecific?.parsedData?.name?.toLowerCase();
```

Comment:
can it be simplified somehow?

Reply:
refactored https://github.com/trezor/trezor-suite/pull/12538/commits/d874f631c0ab6ad1cd52a7228cd1da9de192a84d